hws/src/dr_ste.py

```diff
@@ -288,6 +288,22 @@ def dump_fields(self, verbosity, tabs):
 
         return _str
 
+    def dump_miss(self, verbosity, tabs, expected_miss_index):
+        if not len(self.fields_dic):
+            return ''
+
+        miss_is_expected_loc = (
+            self.miss_loc.gvmi_str == _config_args.get("vhca_id")
+            and
+            hex(self.miss_loc.index) == expected_miss_index
+        )
+        if miss_is_expected_loc and verbosity < 3:
+             return ''
+
+        expected_str = 'is' if miss_is_expected_loc else 'NOT'
+        return tabs + 'Miss: ' + str(self.miss_loc) + \
```

Comment:
change to miss address
and in case of expected no need for "(is expected value)"

Reply:
> change to miss address

This deliberately prints the location rather than just the address/index, so that if the miss is in a different GVMI, the GVMI field will be printed. Note that when the location's GVMI matches the dump's GVMI, the GVMI is not printed, so only the index field gets printed.

Reply:
> change to miss address

Oh, did this mean just change "Miss" to "Miss address"? I thought you were talking about the value that was printed, not the text...